src/utils/vcs.rs

```diff
@@ -569,13 +569,30 @@ pub fn find_head_sha() -> Result<String> {
     Ok(head.id().to_string())
 }
 
-pub fn find_base_sha() -> Result<Option<String>> {
-    let github_event = std::env::var("GITHUB_EVENT_PATH")
-        .map_err(Error::from)
-        .and_then(|event_path| std::fs::read_to_string(event_path).map_err(Error::from))
-        .context("Failed to read GitHub event path")?;
+pub fn find_base_sha(remote_name: &str) -> Result<Option<String>> {
```

Comment:
As we no longer ever return `Ok(None)` from this function, we should simplify the signature to return `Result<String>`.

```suggestion
pub fn find_base_sha(remote_name: &str) -> Result<String> {
```

Reply:
Thanks!